homeassistant/components/izone/config_flow.py

```diff
@@ -16,11 +16,11 @@
 
 
 async def _async_has_devices(hass):
+    async def dispatch_discovered(_):
```

Comment:
This can be a callback since we don't await inside.
```suggestion
    @callback 
    def dispatch_discovered(_):
```

Reply:
Fixed